control/control.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python
+#
+# Copyright (c) 2019, CESAR. All rights reserved.
+#
+# SPDX-License-Identifier: BSD 3-Clause
+
+import dbus
+import dbus.service
+import dbus.mainloop.glib
+import os
+
+from reset_factory import ResetFactory
+from dbus.exceptions import DBusException
+
+class Control(dbus.service.Object):
+    """
+    Control KNoT Gateway state using a dbus implementation
+    """
+
+    def __init__(self):
+        self.reset = ResetFactory.create('settings.json')
```

Comment:
Please, do not pass this parameter. Use it inside the class as a constant. Also, rename it to something more specific.